invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/__tests__/__snapshots__/ItemsSearch.js.snap

```diff
@@ -91,23 +75,25 @@ exports[`PatronLoans tests should load the ItemsSearch component 1`] = `
 `;
 
 exports[`PatronLoans tests should render items in search 1`] = `
-<ItemsCheckout
+<ItemsSearch
   checkoutItem={[MockFunction]}
   clearResults={[MockFunction]}
   fetchItems={[MockFunction]}
   fetchPatronCurrentLoans={[MockFunction]}
   history={[Function]}
   items={
-    Array [
-      Object {
-        "barcode": "3",
-        "document_pid": 4,
-        "item_pid": "4",
-        "status": "LOANABLE",
-      },
-    ]
+    Object {
+      "hits": Array [
+        Object {
+          "barcode": "3",
+          "document_pid": 4,
```

Comment:
And here strings for the win, and I am going to stop flooding the review with string comments, so it goes for everything further down.